tests/utils.go

```diff
@@ -4335,10 +4351,23 @@ func SkipIfARM64(message string) {
 	}
 }
 
+func VolumeExpansionAllowed(sc string) bool {
+	virtClient, err := kubecli.GetKubevirtClient()
+	Expect(err).ToNot(HaveOccurred())
+	storageClass, err := virtClient.StorageV1().StorageClasses().Get(context.Background(), sc, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	return storageClass.AllowVolumeExpansion != nil &&
+		*storageClass.AllowVolumeExpansion
+}
+
 func HasBindingModeWaitForFirstConsumer() bool {
 	virtClient, err := kubecli.GetKubevirtClient()
 	Expect(err).ToNot(HaveOccurred())
-	storageClass, err := virtClient.StorageV1().StorageClasses().Get(context.Background(), Config.StorageClassLocal, metav1.GetOptions{})
+	sc, exists := GetRWOFileSystemStorageClass()
```

Comment:
So for the existing test cases, I think all the WaitForFirstConsumer ones use Filesystem volume mode. But it is entirely possilbe to have block WaitForFirstConsumer, for instance https://github.com/topolvm/topolvm is such a provisioner that provides block WaitForFirstConsumer. Nothing to change right now, just be aware it is possible.

Reply:
as I wrote in a previous commit as I can add snapshotStorage and expansionStorage I can also add WFFCStorage..

Reply:
This was more just an FYI, this function checks for WFFC, and it assumes it is filesystem volume mode, I was pointing out it is possible to have WFFC with block volume mode.